Lesson-11/Lesson-11.html

```diff
@@ -0,0 +1,134 @@
+	<title>First steps</title>
+</head>
+
+<body>
+	<!--The main wrapper of the first page of the project-->
+	<div class="wrapper">
+		<!--navigation and company logo-->
+		<header class="header_logo_navigation">
+			<img class="header_company logo" src="LinkedSprite.svg#src-2" alt="Company `s logo">
+			<!--header navigation-->
+			<nav class="header_navigation">
+				<a href="#" class="header_link link_stays">Stays</a>
+				<a href="#" class="header_link link_attractions">Attractions</a>
+				<a href="#" class="header_link night_icon">
+					<img class="header_icon night_icon" src="LinkedSprite.svg#src-3">
+				</a>
+				<a href="#" class="header_link circle_icon ">
+					<img class="header_icon circle_icon " src="LinkedSprite.svg#src-4">
+				</a>
+				<!--burger menu-->
+				<a class="header_burger"><span></span></a>
+			</nav>
+		</header>
+		<section class="section_search">
+			<h1>Discover stays
+				<br>
+				to live, work or just relax</h1>
+			<div class="section_label">
+				<label for="input_hotel" class="label">Your destination or hotel name</label>
+				<label for="input_date" class="label">Check-in — Check-out</label>
+			</div>
+			<form action="#" method="POST" class="search-form">
+				<input class="input-text" name="input-text" type="text" id="input_hotel" placeholder="New York">
+				<input class="input-data" type="date" id="input_date">
+				<p id="adultsButton"> Adults-<p id="placeholderCounter">0</p>          
+				<p id="childrenButton"> Children-<p id="placeholderChildren"> 0 </p>
+				<p id="roomButton"> Room-<p id="placeholderRoom">0</p>
+				<input class="submit" type="submit" value="Search">
+			</form>
+			
+			<div class="section_icon">
+				<a href="#"><img class="google-store-icon" src="LinkedSprite.svg#src-1"></a>
+				<a href="#"><img class="apple-icon" src="LinkedSprite.svg#src-5"></a>
+			</div>
+			
+		</section>
+	</div>
+
+		<div class="filter-form" id="filterForm">
+			<div class="item_buttons">
+				 <span>Adults</span>
+				 <div class="buttons"> 
+					  <button  class="counter-minus" id="minus-adults" >-</button>
+					  <div class="counter"  id="counterAdult" >0</div>
+					  <button class="counter-plus " id="plus-adults">+</button>
+				 </div>
+			</div>
+			<div class="item_buttons">
+			 <span>Children</span>
+			 <div class="buttons"> 
+				  <button class="counter-minus" id="minus-children" onclick="remove()">-</button>
+				  <div class="counter" id="counterChildren">0</div>
+				  <button class="counter-plus" id="plus-children" >+</button>
+			 </div>
+		</div>
+		<div class="item_buttons">
+			 <span>Rooms</span>
+			 <div class="buttons">                                           
+				  <button  class="counter-minus" id="RoomsMinus">-</button>
+				  <div class="counter"  id="counterRooms">0</div>
+				  <button class="counter-plus" id="plus-rooms">+</button>
+			 </div>
+
+
+		</div>
+
+
+		     <p  id="age_select">What is the age of the child you’re travelling with?</p>
```

Comment:
unnecessary space after reducing amount of children
![image](https://user-images.githubusercontent.com/20189313/141701510-7edeb3b7-5707-4d33-8249-f486e0470a8e.png)
Ans add space in input